mne/report/tests/test_report.py

```diff
@@ -264,14 +266,17 @@ def test_report_raw_psd_and_date(tmpdir):
 
     tempdir = str(tmpdir)
     raw = read_raw_fif(raw_fname).crop(0, 1.).load_data()
+    raw.info['experimenter'] = 'mne test'
+    raw.info['subject_info'] = dict(id=123, his_id='sample')
+
     raw_fname_new = op.join(tempdir, 'temp_raw.fif')
     raw.save(raw_fname_new)
     report = Report(raw_psd=True)
     report.parse_folder(data_path=tempdir, render_bem=False,
                         on_error='raise')
     assert isinstance(report.html, list)
     assert 'PSD' in ''.join(report.html)
-    assert 'GMT' in ''.join(report.html)
```

Comment:
Seems like a backward incompat change to go from what used to be GMT to something else. -1 on going to the current computer's time (this makes my result and your result differ when analyzing the same data), but +1 on going to the acquisition machine's time zone if available (can't remember if this is stored or not).

If we want, we can use JavaScript to convert to the viewer's time zone on the fly like we do with out dev doc footer:

https://github.com/mne-tools/mne-python/blob/main/doc/conf.py#L53-L55

This makes the contents of the file the same, but when viewed in a browser, it shows local.

Reply:
Actually this line was removed by accident, we don't have a regression here. I added the test back, no further changes required.